src/components/UI/NetworkBar/NetworkBar.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { useState } from "react";
+import { NETWORK } from "../../../config";
+import styled from "@emotion/styled";
+import { vars } from "../../../styles";
+import { SvgIcon, SvgIconX } from "../SvgIcon";
+
+const NetworkWrap = styled.div`
+  background-color: #001f29;
```

Comment:
Same thing, the color should be in the vars hor.

Reply:
ok